src/recipes.jl

```diff
@@ -5,14 +5,14 @@
 
 dateformatter(dt) = string(Date(Dates.UTD(dt)))
 datetimeformatter(dt) = string(DateTime(Dates.UTM(dt)))
-timeformatter(t) = string(Dates.Time(Dates.Nanosecond(t)))
+timeformatter(t) = string(Dates.Time(Dates.Nanosecond(round(t))))
```

Comment:
```suggestion
timeformatter(t) = string(Dates.Time(Dates.Nanosecond(round(Int, t))))
```
just to make it explicit that these have to be `Int`s.